src/test/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtilsUnitTest.java

```diff
@@ -5,28 +5,55 @@
 import org.apache.log4j.Logger;
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
-import org.testng.Assert;
-
-
-import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.*;
-import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.SequenceDictionaryCompatibility.*;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-public class SequenceDictionaryUtilsUnitTest extends BaseTest {
+import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.*;
+import static org.broadinstitute.hellbender.utils.SequenceDictionaryUtils.SequenceDictionaryCompatibility.*;
+
+public final class SequenceDictionaryUtilsUnitTest extends BaseTest {
 
     private static Logger logger = Logger.getLogger(SequenceDictionaryUtilsUnitTest.class);
 
+    @DataProvider( name = "SAMSequenceRecords" )
+    public Object[][] SAMSequenceRecords() {
```

Comment:
For `DataProviders` used by only 1 test case, like this one, the name should reflect the test case instead of having a super-generic name like `SAMSequenceRecords` -- eg., something like `SequenceRecordsAreEquivalentData`. Otherwise the class becomes confusing (eg., there is a "SequenceDictionaryDataProvider" below used by multiple test cases).